src/components/ui/Contact/Contact.tsx

```diff
@@ -1,10 +1,40 @@
+import { cva } from 'class-variance-authority';
 import { LocalContact } from '../../../types/LocalTypes';
+import classNames from 'classnames';
 
-const Contact = ({ name, email }: LocalContact) => {
+const nameVariants = cva('', {
+  variants: {
+    size: {
+      small: ['text-13px'],
+      large: ['text-[1.25rem]', 'font-semibold'],
+    },
+  },
+  defaultVariants: {
+    size: 'small',
+  },
+});
+
+const emailVariants = cva('text-greyText', {
+  variants: {
+    size: {
+      small: ['text-xs'],
+      large: ['text-sm'],
+    },
+  },
+  defaultVariants: {
+    size: 'small',
+  },
+});
+
+interface ContactProps extends LocalContact {
+  size: 'small' | 'large';
+}
```

Comment:
I think you could extract the size variant using a helper that comes with the library.

See more here: https://cva.style/docs/getting-started#variantprops